nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>This section doesn't show how to install it, please remove it or add a link to <a href="https://nixtlaverse.nixtla.io/neuralforecast/examples/installation.html" target="_blank">https://nixtlaverse.nixtla.io/neuralforecast/examples/installation.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added link as comment.</p>